Frameworks/CoreGraphics/CGImage.mm

```diff
@@ -376,6 +369,54 @@ CGImageRef CGImageCreateWithMaskingColors(CGImageRef image, const CGFloat* compo
 
 #pragma region WIC_HELPERS
 
+DisplayTexture* _CGImageGetDisplayTexture(CGImageRef image) {
+    RETURN_NULL_IF(!image);
+    CGIWICBitmap* iwicImg = dynamic_cast<CGIWICBitmap*>(image->ImageSource().Get());
```

Comment:
We should be keeping these all within ComPtr rather than mucking around with raw pointers